httpretty/core.py

```diff
@@ -1130,7 +1130,7 @@ def fill_filekind(self, fk):
             )
 
         for item in string_list:
-            fk.write(utf8(item) + b'\n')
+            fk.write(utf8(item) + b'\r\n')
```

Comment:
Also, UTF-8 is not valid in headers:

> Field values are usually constrained to the range of US-ASCII characters [[USASCII](https://www.rfc-editor.org/rfc/rfc9110.html#USASCII)]. Fields needing a greater range of characters can use an encoding, such as the one defined in [[RFC8187](https://www.rfc-editor.org/rfc/rfc9110.html#RFC8187)].
https://www.rfc-editor.org/rfc/rfc9110.html#section-5.5-4

> A recipient MUST parse an HTTP message as a sequence of octets in an encoding that is a superset of US-ASCII [[USASCII](https://www.rfc-editor.org/rfc/rfc9112.html#USASCII)]. Parsing an HTTP message as a stream of Unicode characters, without regard for the specific encoding, creates security vulnerabilities due to the varying ways that string processing libraries handle invalid multibyte character sequences that contain the octet LF (%x0A).
https://www.rfc-editor.org/rfc/rfc9112.html#section-2.2-2